docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -261,6 +261,25 @@ In general the recommended workaround is to set the following environment variab
         $ export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
 
 
+.. _macos_as_a_target_faq:
+
+Running on macOS as a target
+----------------------------
+
+When managing a system with macOS Monterey 12, macOS Ventura
+13 or above over SSH, one might encounter the following error:
```

Comment:
```suggestion
When managing a system with macOS Monterey 12, macOS Ventura
13 or above over SSH, the following error can occur:
```

Reply:
Just a style thing but I think the more formal tone can be a little distracting.